Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "GCPIAMDataConnector",
+    "title": "Google Cloud Platform IAM (Preview)",
+    "publisher": "Google Cloud Platform",
```

Comment:
Can you tell us whether the Publisher is "Google Cloud Platform" or "Google"? For 'GoogleWorkspaceReportsAPI', publisher was 'Google', hence asking.

Reply:
I think it is better to use "Google" here. File updated. Thanks